inquire/src/prompts/path_select/mod.rs

```diff
@@ -0,0 +1,365 @@
+
+    /// Whether to show hidden files.
+    pub show_hidden: bool,
+
+    /// Whether to show symlinks
+    pub show_symlinks: bool,
+
+    /// [How to sort](SortingMode) listed files and directories
+    pub sorting_mode: PathSortingMode,
+
+    /// Whether to allow selecting multiple files
+    pub select_multiple: bool,
+
+    /// The divider to use in the selection list following current-directory entries
+    pub divider: &'a str,
+
+    /// Function that formats the user input and presents it to the user as the final rendering of the prompt.
+    pub formatter: MultiOptionFormatter<'a, PathEntry>,
+
+    /// Whether the current filter typed by the user is kept or cleaned after a selection is made.
+    pub keep_filter: bool,
+
+    /// RenderConfig to apply to the rendered interface.
+    ///
+    /// Note: The default render config considers if the NO_COLOR environment variable
+    /// is set to decide whether to render the colored config or the empty one.
+    ///
+    /// When overriding the config in a prompt, NO_COLOR is no longer considered and your
+    /// config is treated as the only source of truth. If you want to customize colors
+    /// and still suport NO_COLOR, you will have to do this on your end.
+    pub render_config: RenderConfig<'a>,
+    /// The [path selection mode](PathSelectionMode) determines what the user can select.
+    pub selection_mode: PathSelectionMode<'a>,
+}
+
+impl<'a, T> PathSelect<'a, T>
+where
+    T: AsRef<Path>,
+{
+    /// PathEntry formatter used by default in [PathSelect](crate::PathSelect) prompts.
+    /// Prints the string value of all selected options, separated by commas.
+    ///
+    /// See [PathSelect::DEFAULT_PATH_FORMATTER]
+    pub const DEFAULT_FORMATTER: MultiOptionFormatter<'a, PathEntry> =
+        &|ans| PathSelect::<PathEntry>::DEFAULT_PATH_FORMATTER(ans);
+
+    /// Path formatter used by default in [PathSelect](crate::PathSelect) prompts.
+    /// Prints the string value of all selected options, separated by commas.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    ///
+    /// use inquire::list_option::ListOption;
+    /// use inquire::{PathSelect, PathEntry};
+    /// use std::{fs::FileType, path::PathBuf};
+    ///
+    /// let formatter = PathSelect::<PathBuf>::DEFAULT_PATH_FORMATTER;
+    /// let a = PathBuf::from("/ra/set/nefer.rs");
+    /// let mut ans = vec![ListOption::new(0, &a)];
+    /// assert_eq!(String::from("/ra/set/nefer.rs"), formatter(ans.as_slice()));
+    ///
+    /// let b = PathBuf::from("/maat/nut.rs");
+    /// ans.push(ListOption::new(3, &b));
+    /// assert_eq!(String::from("/ra/set/nefer.rs, /maat/nut.rs"), formatter(ans.as_slice()));
+    ///
+    /// let c = PathBuf::from("ptah.rs");
+    /// ans.push(ListOption::new(7, &c));
+    /// assert_eq!(String::from("/ra/set/nefer.rs, /maat/nut.rs, ptah.rs"), formatter(ans.as_slice()));
+    /// ```
+
+    pub const DEFAULT_PATH_FORMATTER: MultiOptionFormatter<'a, T> = &|ans| {
```

Comment:
```suggestion
    /// ```
    pub const DEFAULT_PATH_FORMATTER: MultiOptionFormatter<'a, T> = &|ans| {
```